### DISCUSSION TITLE
How to make grid static-PorousFlow

### DISCUSSION THREAD

**Discussion by rksin8** [idaholab/moose#20789] [Q&A Modules: General] [answered]
Hi 
I am using PorousFlow module and solving a single-phase hydromechanical problem. Grid dimension increases in each time step with the fluid injection. I want to fix the gird for the whole simulation time. I have used `use_displaced_mesh = false` but still no effect on grid dimension. 

Geometry at t=0: xmax=200
![image](https://user-images.githubusercontent.com/22324848/163389769-59cce978-c9dd-4f86-bb73-b98d1555b383.png)

Geometry at some other time: xmax=280
![image](https://user-images.githubusercontent.com/22324848/163389935-27423e48-0bae-4dbd-a52b-c5ce0f8a6d93.png)

Here is my [Input file](https://github.com/rksin8/moose_test/blob/main/quasi_2d.i)

Thank you!

**Comment by josebastiase** [comment]
Hi, 
Maybe I got you need to fix the displacement at the boundaries. Try this

```
  [no_x_disp]
    type = DirichletBC
    variable = disp_x
    value = 0
    boundary = 'left right'
  []
  [no_y_disp]
    type = DirichletBC
    variable = disp_z
    value = 0
    boundary = 'bottom top'
  []
```

Cheers

**Reply by rksin8** [reply]
Thank you for your reply. I am working on a benchmark problem [5.1.1Test Case 1](https://link.springer.com/article/10.1007/s10596-020-09951-8) where the right face is allowed to move freely. 
![image](https://user-images.githubusercontent.com/22324848/163899877-f83b8ffe-5927-46e1-9088-e10f14e700b8.png)

I always struggle when I validate published problems. I think Young's modules for sandstone (10MPa) used is too less, which results in high displacements.

**Comment by cpgr** [comment, marked as answer]
Without actually running this to check, I think that you are injecting ~1000 times more than you think as you are using `multiply_by_density = false` (so fluxes are volumetric, not mass) in the kernels but then injecting a mass flux, which is a factor of water density greater than you want. Commenting out the `multiply_by_density` parameters should correct this.

**Comment by cpgr** [comment]
Also, thanks for the paper with these benchmark problems. When you get this working, these would make nice examples to add to the documentation if you like.
